src/test/java/com/rackspace/salus/monitor_management/services/MonitorManagementTest.java

```diff
@@ -1528,11 +1526,12 @@ public void testRemoveMonitor() {
             .setLabelSelectorMethod(LabelSelectorMethod.AND)
             .setInterval(Duration.ofSeconds(60)));
 
-    final BoundMonitorProjection boundMonitor = new BoundMonitorProjection("e-goner", monitor, "t-1", "r-1", "zoneA");
-
-    when(boundMonitorRepository.findAllByTenantIdAndMonitor_IdIn(anyString(), any()))
-        .thenReturn(Collections.singletonList(boundMonitor));
-
+    BoundMonitor b = new BoundMonitor();
+    b.setMonitor(monitor);
+    b.setResourceId("r-1");
+    b.setEnvoyId("e-goner");
+    when(boundMonitorRepository.findAllByTenantIdAndMonitor_IdIn(anyString(), any(), any()))
+        .thenReturn(new PageImpl<>(Collections.singletonList(b))).thenReturn(Page.empty());
```

Comment:
```suggestion
    when(boundMonitorRepository.findAllByTenantIdAndMonitor_IdIn(anyString(), any(), any()))
        .thenReturn(new PageImpl<>(Collections.singletonList(b)))
        .thenReturn(Page.empty());
```

Can you switch it to this so its a little clearer that multiple returns are being used.